specification/vmware/Microsoft.AVS.Management/client.tsp

```diff
@@ -0,0 +1,105 @@
+import "@azure-tools/typespec-client-generator-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+import "./main.tsp";
+
+using Azure.ClientGenerator.Core;
+using Microsoft.AVS;
+
+// flatten
+#suppress "deprecated" "property flatten for SDK backward compatibility"
+@@flattenProperty(Azure.ResourceManager.TrackedResource.properties);
```

Comment:
You should just turn on flattening for the spec rather than having all of these.

Reply:
I assume the flattening on the spec only apply to the Swagger output? Does it add `@flattenProperty` decorate (which is the only one recognized by TCGC).